hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/fs/FSBuilder.java

```diff
@@ -61,6 +61,13 @@
    */
   B opt(@Nonnull String key, float value);
 
+  /**
+   * Set optional long parameter for the Builder.
+   *
+   * @see #opt(String, String)
+   */
+  B opt(@Nonnull String key, long value);
```

Comment:
We could use properties (getter and setter) for the standard options.  We'd have one for buffer size, one for file length, etc.